packages/cactus-plugin-odap-hermes/src/test/typescript/integration/odap/transfer-complete.test.ts

```diff
@@ -44,8 +44,7 @@ test("dummy test for transfer complete flow", async (t: Test) => {
     JSON.stringify(transferCompleteReq),
     dummyPrivKeyStr,
   );
-  t.doesNotThrow(
+  expect(
```

Comment:
@awadhana Sorry, my previous advice was not 100% accurate. :/ 

It looks perfect, BUT could you also add an `await` here before the `expect(...)` call so that the test definitely waits for the assertion to finish? (my fault for missing this earlier, apologies) 

I figured out my mistake by looking at the JSdoc of the expect function via VSCode and seeing that it returns a promise instead of `void` or anything else really (just a useful tip on how to discover similar info in other contexts):

![Screenshot from 2022-01-27 12-37-09](https://user-images.githubusercontent.com/1583231/151440002-2240ec35-6430-4323-a94f-3bd8b9848601.png)